packages/web-api/src/methods.ts

```diff
@@ -62,48 +62,95 @@ export interface AdminAppsApproveArguments extends WebAPICallOptions, TokenOverr
   team_id?: string;
 }
+cursorPaginationEnabledMethods.add('admin.apps.requests.list');
 export interface AdminAppsRestrictArguments extends WebAPICallOptions, TokenOverridable {
   app_id?: string;
   request_id?: string;
   team_id?: string;
 }
+export interface AdminAppsRestrictedListArguments extends WebAPICallOptions, TokenOverridable, CursorPaginationEnabled {
+  team_id?: string;
+  enterprise_id?: string;
+}
+cursorPaginationEnabledMethods.add('admin.apps.restricted.list');
+export interface AdminConversationsSetTeamsArguments extends WebAPICallOptions, TokenOverridable {
+  channel_id: string;
+  team_id?: string;
+  target_team_ids?: string[];
+  org_channel?: boolean;
+}
 export interface AdminInviteRequestsApproveArguments
   extends WebAPICallOptions, TokenOverridable {
   invite_request_id: string;
   team_id: string;
 }
+export interface AdminInviteRequestsApprovedListArguments
+  extends WebAPICallOptions, TokenOverridable, CursorPaginationEnabled {
+  team_id: string;
+}
+cursorPaginationEnabledMethods.add('admin.inviteRequests.approved.list');
 export interface AdminInviteRequestsDenyArguments
   extends WebAPICallOptions, TokenOverridable {
   invite_request_id: string;
   team_id: string;
 }
-export interface AdminInviteRequestsListArguments
-  extends WebAPICallOptions, TokenOverridable, CursorPaginationEnabled {
-  team_id: string;
-}
-export interface AdminInviteRequestsApprovedListArguments
+export interface AdminInviteRequestsDeniedListArguments
   extends WebAPICallOptions, TokenOverridable, CursorPaginationEnabled {
   team_id: string;
 }
-export interface AdminInviteRequestsDeniedListArguments
+cursorPaginationEnabledMethods.add('admin.inviteRequests.denied.list');
+export interface AdminInviteRequestsListArguments
   extends WebAPICallOptions, TokenOverridable, CursorPaginationEnabled {
   team_id: string;
 }
+cursorPaginationEnabledMethods.add('admin.inviteRequests.list');
 export interface AdminTeamsAdminsListArguments extends WebAPICallOptions, TokenOverridable, CursorPaginationEnabled {
   team_id: string;
 }
+cursorPaginationEnabledMethods.add('admin.teams.admins.list');
 export interface AdminTeamsCreateArguments extends WebAPICallOptions, TokenOverridable {
   team_domain: string;
   team_name: string;
   team_description?: string;
   team_discoverability?: string;
 }
+export interface AdminTeamsListArguments extends WebAPICallOptions, TokenOverridable, CursorPaginationEnabled {}
+cursorPaginationEnabledMethods.add('admin.teams.list');
 export interface AdminTeamsOwnersListArguments extends WebAPICallOptions, TokenOverridable, CursorPaginationEnabled {
   team_id: string;
 }
+cursorPaginationEnabledMethods.add('admin.teams.owners.list');
+export interface AdminTeamsSettingsInfoArguments extends WebAPICallOptions, TokenOverridable {
+  team_id: string;
+}
+export interface AdminTeamsSettingsSetDefaultChannelsArguments extends WebAPICallOptions, TokenOverridable {
+  team_id: string;
+  channel_ids: string[];
```

Comment:
needs to be `string` as with `target_team_ids`

Reply:
(same as above)